webui/src/pages/extension-detail/extension-detail.tsx

```diff
@@ -210,53 +210,53 @@ export class ExtensionDetailComponent extends React.Component<ExtensionDetailCom
+        return (
+            <Box overflow='auto'>
+                <Typography variant='h5' className={classes.titleRow}>
+                    {extension.displayName || extension.name} {extension.preview ?
+                        <span className={`${classes.preview} ${themeClass}`}>preview</span>
+                        : ''}
+                </Typography>
+                <Box className={`${themeClass} ${classes.infoRow} ${classes.alignVertically}`}>
+                    {this.renderAccessInfo(extension, themeClass)}&nbsp;<span
+                        title='Unique identifier'
+                        className={classes.code}>
+                        {extension.namespace}.{extension.name}
+                    </span>
+                    <TextDivider theme={theme} />
+                    Published by <Link href={extension.publishedBy.homepage}
+                        className={`${classes.link} ${themeClass}`}>
+                        {
+                            extension.publishedBy.avatarUrl ?
+                                <React.Fragment>
+                                    {extension.publishedBy.loginName}&nbsp;<Avatar
+                                        src={extension.publishedBy.avatarUrl}
+                                        alt={extension.publishedBy.loginName}
+                                        variant='circle'
+                                        classes={{ root: classes.avatar }} />
+                                </React.Fragment>
+                                : extension.publishedBy.loginName
+                        }
+                    </Link>
+                    <TextDivider theme={theme} />
+                    {this.renderLicense(extension, themeClass)}
+                </Box>
+                <Box mt={2} mb={2} overflow='auto'>
+                    <Typography classes={{ root: classes.description }}>{extension.description}</Typography>
+                </Box>
+                <Box className={`${themeClass} ${classes.infoRow} ${classes.alignVertically}`}>
+                    <SaveAltIcon fontSize='small' />&nbsp;{extension.downloadCount || 0}&nbsp;{extension.downloadCount === 1 ? 'download' : 'downloads'}
+                    <TextDivider theme={theme} />
+                    <RouteLink
+                        to={createRoute([ExtensionDetailRoutes.ROOT, extension.namespace, extension.name, ExtensionDetailRoutes.TAB_REVIEWS])}
+                        className={`${classes.link} ${themeClass} ${classes.alignVertically}`}
+                        title={
+                            extension.averageRating !== undefined ?
+                                `Average rating: ${this.getRoundedRating(extension.averageRating)} out of 5`
+                                : 'Not rated yet'
+                        }>
+                        <ExportRatingStars number={extension.averageRating || 0} fontSize='small' />
+                        ({extension.reviewCount})
```

Comment:
Please revert the formatting changes here. That would conflict with what I've been working on in the `privileged-publish` branch.